docs/source/tutorial_mturk.rst

```diff
@@ -166,6 +166,48 @@ You can programmatically review work using the commands available in the `MTurkM
 If you don't take any action in 4 weeks, all HITs will be auto-approved and Turkers will be paid.
 
 
+ParlAI-MTurk Tips and Tricks
+----------------------------
+
+Approving Work
+^^^^^^^^^^^^^^
+
+- Unless you explicitly set the flag `—auto-approve-delay` or approve the agents work by calling `mturk_agent.approve_work()`, work will be auto approved after 30 days; workers generally like getting paid sooner than this so set the `auto_approve_delay` to be shorter when possible
+- Occasionally Turkers will take advantage of getting paid immediately without review if you auto approve their work by calling `mturk_agent.approve_work()` at the close of the task. If you aren't using any kind of validation before you `approve_work` or if you don't intend to review the work manually, consider setting the `—auto-approve-delay` flag rather than approving immediately.
+
+Rejecting Work
+^^^^^^^^^^^^^^
+
+- Most Turkers take their work very seriously, so if you find yourself with many different workers making similar mistakes on your task, it's possible the task itself is unclear. You **shouldn't** be rejecting work in this case, rather you should update your instructions and see if the problem resolves.
+- Reject sparingly at first and give clear reasons for rejection/how to improve. Rejections with no context are a violation of Amazon's TOS
+
+Soft-blocking vs. Hard-blocking
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Hard block sparingly; it's possible workers that aren't doing well on a particular task are perfectly good at others. Hard blocking reduces your possible worker pool.
+- Soft blocking workers that are clearly trying on a task but not **quite** getting it allows those workers to work on other tasks for you in the future. You can soft block workers by calling `mturk_manager.soft_block_worker(<worker id>)` after setting `—block-qualification`. That worker will not be able to work on any tasks that use the same `—block-qualification`.
```

Comment:
maybe we should tell people to make the soft block qualification descriptive; it seems turkers like to know *why* they are soft blocked (as it is not always their fault)

Reply:
Also the default message they get is something along the lines of "You've been soft blocked because of too many disconnects" - this is probably something we should change

Reply:
Intending to split the disconnect block into it's own qualification today 👍